Control/public/Model.js

```diff
@@ -297,14 +297,25 @@ export default class Model extends Observable {
   }
 
   /**
-   * After pages load check if the user enabled notifications
+   * Request notification permission
    */
-  checkBrowserNotificationPermissions() {
+  requestBrowserNotificationPermissions() {
     if (Notification.permission !== 'granted') {
       Notification.requestPermission();
     }
   }
 
+  /**
+   * Defines policy when user can trigger request for enabling notifications
+   * This is used to display/hide enable button
+   * @returns {bool} True when notifications are not enabled and HTTPS is used
```

Comment:
`bool` is of type `boolean` in JS
```suggestion
   * @returns {boolean} True when notifications are not enabled and HTTPS is used
```